src/Meshes.jl

```diff
@@ -382,6 +382,7 @@ export
   LambdaMuSmoothing,
   LaplaceSmoothing,
   TaubinSmoothing,
+  Rotation,
```

Comment:
Let's rename it to RotateCoords for consistency with ScaleCoords

Reply:
Are you sure? I don't like it. `ScaleCoords` is nice: this is really a transformation of the coordinates, it depends on the Cartesian coordinates system. But a rotation is a transformation of the points, it does not depend on the Cartesian coordinates system.

Reply:
Can you elaborate on how rotation differs from scaling in this sense? It is not clear to me the distinction you are making.

Reply:
I mean a rotation is not defined as an operation on coordinates. It has a coordinates-free definition. We rotate a point or a vector, not a list of numbers.

Reply:
I see, you are probably referring to the geometric algebra definition? Here we are performing an operations on coordinates though, so our definition literally changes the coordinates, I think it is fine if we use RotateCoords for now. In the future, when we treat geometric algebra more seriously, we can come back and rename some of these transforms.

Reply:
We can use the name RotateCoords temporarily. It is a good name to disambiguate with other transforms. It also has the benefit that end-users will simply ready ScaleCoords, RotateCoords and TranslateCoords and will immediately associate these transforms into the same group. Sometimes we need to be a bit less formal with these names to facilitate the lives of people who are not geometry experts.

Reply:
As you want. But this is a rotated number:
![rfive](https://user-images.githubusercontent.com/4466543/209231273-fddf8e9b-d605-4f0d-965f-1e1b2c5bf60f.png)

Reply:
hahah, good joke. In any case we rotated the coordinates of the associated mesh  and geometries to get that cool rotated 5